### PR TITLE
add full names to filter dropdowns

### PR DESCRIPTION
**Pull request by smk762**
Fixes https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/1287
image

### CODE REVIEW

**Comment by smk762** [comment]
Would adding coin icon here be worth doing also?

**Comment by tonymorony** [comment]
> Would adding coin icon here be worth doing also?

I think it would be visual overkill to add an icon here

File: atomic_defi_design/qml/Components/DexSweetComboBox.qml [hunk 1]
```diff
@@ -150,7 +150,8 @@ ComboBox {
                         highlighted: control.highlightedIndex === index
                         //foregroundColor: DexTheme.foregroundColor
                         contentItem: DefaultText {
-                            text_value: ticker
+                            text_value: "<b><font color='" +DexTheme.getCoinColor(ticker) + "'>" + ticker + "</font></b>" + "    %1".arg(General.coinName(ticker))
```
Comment (SylEze) [inline atomic_defi_design/qml/Components/DexSweetComboBox.qml:153]: Try to add spaces between ops

**Review by SylEze — CHANGES_REQUESTED** [review]

**Review by tonymorony — APPROVED** [review]
lgtm, thanks!

**Pull request merged**